recipes/botan/all/conanfile.py

```diff
@@ -0,0 +1,310 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanException, ConanInvalidConfiguration
+from conans.model.version import Version
+
+
+class BotanConan(ConanFile):
+    name = 'botan'
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/randombit/botan"
+    license = "BSD-2-Clause"
+    exports = ["patches/*"]
+    description = "Botan is a cryptography library written in C++11."
```

Comment:
Add some topics to pass the hook check
```suggestion
    description = "Botan is a cryptography library written in C++11."
    topics = ("crytography", "crypto", "C++11")
```